src/webgl/p5.RendererGL.js

```diff
@@ -383,6 +383,11 @@ p5.RendererGL.prototype.resize = function(w,h) {
   var gl = this.GL;
   p5.Renderer.prototype.resize.call(this, w, h);
   gl.viewport(0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight);
+  // If we're using the default camera, update the aspect ratio
+  if(this._curCamera === 'default') {
+    this._curCamera = null;
```

Comment:
Do we really need to set the _curCamera to null if we are just resetting it to default in the _setDefaultCamera funct?